bin/convert_getty_to_tei

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+CUR=`dirname $0`
+SOURCEDIR=$CUR/../tmp/cicorecords
+TARGETDIR=$CUR/../getty/records
+XSL=$CUR/getty_to_tei.xsl
+java -cp ./saxon-he-10.5.jar net.sf.saxon.Transform -s:$SOURCEDIR -xsl:$XSL -o:$TARGETDIR
```

Comment:
This fails if you try to run it from the parent dir — should this be `-cp $CUR/saxon-he-10.5.jar`?

Reply:
If I try to run this from the `bin` directory, I get an error "Input is a directory, but output is not"

Reply:
So we probably want to make sure the `/getty/records` directory exists — do we want to also checkin the TEI fragment output files?

Reply:
The `getty/records/*.tei.xml` files are intermediary; their teiHeaders are incorporated into the item-level files in `getty/items`.  I think `getty/items` should be checked in, but `getty/records` should not.  That suggests they should live in `tmp`, doesn't it?

Reply:
I've modified the code to write these intermediate files to the local `tmp` directory.